### ISSUE TITLE
Regarding hyperparameters when training the Zero-Shot Evaluation model.

### ISSUE THREAD

**Issue by Yuntian-Gao** [princeton-vl/WAFT#14]
Thank you for sharing your code. I followed the hyperparameters reported in the paper and trained the model for 50k steps each on FlyingChairs and FlyingThings3D, using a learning rate of 1e-4 and a batch size of 32 (with all other hyperparameters set according to the configs in the config/a2/dav2 folder). However, I observed inconsistent performance metrics. Specifically:

Sintel:
Validation clean EPE: 1.3543

Validation final EPE: 2.8946

KITTI:
EPE: 3.8844
1px: 13.3797
Could you please clarify what differences might exist between my reproduction process and yours? Thank you very much!

**Comment by Yuntian-Gao** [comment]
The performance in WAFT:

<img width="484" height="25" alt="Image" src="https://github.com/user-attachments/assets/39b806db-fb54-4891-aa95-bff62c2cbc43" />

**Comment by MemorySlices** [comment]
According to the configs, you should use lr 4e-4 instead of 1e-4 with batch size 32. Also, there might be some variances. You may need to run 2-3 times.
